sql/2019/04_Media/04_01a.sql

```diff
@@ -0,0 +1,105 @@
+#standardSQL
+/*
+04_01a
+
+1.22 TB query
+*Lighthouse Lazyloading, resp images, image quality, webp image score score in 1000 percentile points	
+
+this query takes lighthouse data on image performance: 
+    lazyloading (offscreen), 
+    responsiveness, 
+    quality (optimized),
+    format (webp),
+    animated GIF
+    
+    I break down this data into: 
+         scores (from 0-1) 
+         wasted ms in load time
+         wasted KB
+         count of offending items
+    and break them into percentiles - 26 is the median, but 50 seemed 
+    like a good number to graph the data with
+*/
+
+select 
+      APPROX_QUANTILES( offscreenScore ,1000) offscreenscore,
```

Comment:
You might also want to consider cherry picking a few interesting quantiles rather than returning all 1000. For example, see [this query I wrote](https://github.com/HTTPArchive/almanac.httparchive.org/blob/960965d614ed008955251ce57eb788a9c47fd570/sql/2019/01_JavaScript/01_01a.sql) that picks the 10/25/50/75/90th percentiles out of 1000 quantiles.

Reply:
what I like from 1k quantiles (I had 50 before) is that we can chart the data.  Its a pretty convincing chart to show state of the web -